client/blocks/get-apps/style.scss

```diff
@@ -1,3 +1,6 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 body.get-apps__body {
```

Comment:
Let's drop this entire CSS statement. It gives the body element the wrong background color and the height stuff is already taken care of by other selectors.